### PR TITLE
KAM-37: [FE] Reactive error

### PR DESCRIPTION
**Pull request by HowardRoark96**
- Added `ng-zorro-antd` in `package.json`
- Locked all dependencies` in `package.json`
- Added `custom-bootstrap` styles `_flex`
- Added `COLORS` constant
- Added `app-reactive-error` component

Closes [KAM-37](https://tracker.yandex.ru/KAM-37).

Example view:

- image
- image
- image

### CODE REVIEW

File: src/app/widgets/reactive-fields/reactive-error/reactive-error.component.scss [hunk 1]
```diff
@@ -0,0 +1,10 @@
+@use 'styles/colors' as col;
+
+:host {
+  font-size: 14px;
+  color: col.$red-color-3;
+
+  .anticon {
```
Comment (dim4ik787) [inline src/app/widgets/reactive-fields/reactive-error/reactive-error.component.scss:7]: SuperMinor: excessive nesting

File: src/app/widgets/reactive-fields/reactive-error/reactive-error.component.html [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+    nz-icon
+    nz-tooltip
+    nzType="close-square"
+    nzTheme="twotone"
+    nzTooltipPlacement="left"
+    nzTooltipColor="red"
+    [nzTooltipTitle]="error"
+    [twoToneColor]="COLORS.RED_3"
+  ></span>
+  <span>{{ errorLabel ?? error }}</span>
```
Comment (dim4ik787) [inline src/app/widgets/reactive-fields/reactive-error/reactive-error.component.html:12]: SuperMinor: Mb we can add some default text here?

**Review by dim4ik787 — COMMENTED** [review]
What branch are we branching from? we haven’t frozen that one yet, but this one already has that code.

**Comment by HowardRoark96** [comment]
> What branch are we branching from? we haven’t frozen that one yet, but this one already has that code.

First of all we need to merge `KAM-35` branch into `dev`. Then we need to pull `dev` into current branch to separate code in this `PR`. I needs the code from `KAM-35` here but you couldn't reviwe `KAM-35` branch earler.
FYI. GitHub compares all code with `dev` branch.

image

We need like that:
image

**Pull request merged**